config/schemas/clientSchema.js

```diff
@@ -0,0 +1,8 @@
+import * as z from "zod";
+
+const envSchema = z.object({
+  NEXT_PUBLIC_GA_MEASUREMENT_ID: z.string().optional(),
```

Comment:
Actually I think this is no longer needed - so we can remove all reference to it

Reply:
ok, good to know. thanks!